### PR TITLE
Console command + readme improvements

### PR DESCRIPTION
**Pull request by jurgenwerk**
Added the `yarn console` for convenience and a couple of examples to help with app debugging and testing.

### CODE REVIEW

File: packages/hub/package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
   },
   "scripts": {
     "start": "node ./bin/server.ts",
+    "console": "node --experimental-repl-await ./bin/console.ts",
```
Comment (jurgenwerk) [inline packages/hub/package.json:74]: `--experimental-repl-await` flag allows await to be used "freely", without the async function

**Review by lukemelia — APPROVED** [review]
Nice!

Note: A yarn script for this also exists in the monorepo root-level package.json (https://github.com/cardstack/cardstack/blob/main/package.json#L61). I'm not sure if there is an elegant way to manage yarn scripts at each level.

**Comment by jurgenwerk** [comment]
Oh, I see, didn't check there. I suppose it doesn't hurt to have this on the hub level as well.

**Review by Aierie — APPROVED** [review]
👍

**Review by burieberry — APPROVED** [review]

**Review by backspace — APPROVED** [review]

**Pull request merged**